crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -24,11 +25,39 @@ inherit .enclosing_def
   ;; This provides all the exported symbols from the file
   node @source_unit.defs
 
+  ;; Connect to ROOT_NODE to export our symbols
   node export
   attr (export) pop_symbol = FILE_PATH
   edge ROOT_NODE -> export
   edge export -> @source_unit.defs
 
+  ;; Automatically import language built-ins
+  if (not (eq FILE_PATH BUILT_INS_FILE_PATH)) {
+    ;; All built-in symbols are defined in an internal contract named '$$'
+    ;; so we need to construct an equivalent import path to reach them.
+    ;; We should have access to both type members (eg. defined enums & structs)
+    ;; as well as functions and state variables (see special case below), hence
+    ;; why we're introducing a path through `@typeof`.
+    node built_ins
+    attr (built_ins) push_symbol = BUILT_INS_FILE_PATH
+
+    node built_in_library
+    attr (built_in_library) push_symbol = "$$"
```

Comment:
nit: I suggest adding markers so that they stay in sync. And we can add that everywhere else its name is used/defined.

```suggestion
    ;; All built-in symbols are defined in an internal contract (see __SLANG_BUILT_INS_CONTRACT_NAME__)
    ;; so we need to construct an equivalent import path to reach them.
    ;; We should have access to both type members (eg. defined enums & structs)
    ;; as well as functions and state variables (see special case below), hence
    ;; why we're introducing a path through `@typeof`.
    node built_ins
    attr (built_ins) push_symbol = BUILT_INS_FILE_PATH

    node built_in_library
    attr (built_in_library) push_symbol = "$$" ;; __SLANG_BUILT_INS_CONTRACT_NAME__ (keep in sync)
```